### PR TITLE
Added some links to private anime specific torrent websites 

### PR DESCRIPTION
**Pull request by tanmaymohan**
Links updated and added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,5 +98,19 @@ A curated list of anime streaming/downloading websites.
 * [mangastream.com](http://mangastream.com/)
 * [zingbox.me](http://www.zingbox.me/)
 
+
+<strong>##Private Torrent Trackers</strong>
+* [animebytes.tv](http://animebytes.tv)
```
Comment (anshumanv) [inline README.md]: Why does it require a login ?

File: README.md [hunk 2]
```diff
@@ -98,5 +98,19 @@ A curated list of anime streaming/downloading websites.
...
+* [monoke-bt.org](http://mononoke-bt.org)
```
Comment (anshumanv) [inline README.md]: Please don't put websites that require registration.

File: README.md [hunk 3]
```diff
@@ -98,5 +98,19 @@ A curated list of anime streaming/downloading websites.
...
+* [unionfansub.com](http://unionfansub.com)
```
Comment (anshumanv) [inline README.md]: Dead link

File: README.md [hunk 4]
```diff
@@ -98,5 +98,19 @@ A curated list of anime streaming/downloading websites.
...
+* [tracker.tekeremata.org](http://tracker.tekeremata.org)
```
Comment (anshumanv) [inline README.md]: Not in english.

File: README.md [hunk 5]
```diff
@@ -98,5 +98,19 @@ A curated list of anime streaming/downloading websites.
...
+* [bt.omda-fansubs.com](http://bt.omda-fansubs.com)
```
Comment (anshumanv) [inline README.md]: Dead link

**Comment by tanmaymohan** [comment]
That is why I marked them as private tracker sites.

**Pull request closed without merging**